### ISSUE TITLE
Update version of MySql.data.dll to support mysql_clear_password plugin

### ISSUE THREAD

**Issue by twerthi** [mithrandyr/SimplySql#57]
[AWS RDS IAM role authentication for MySQL](https://aws.amazon.com/premiumsupport/knowledge-center/users-connect-rds-iam/) requires the use of the mysql_clear_password plugin.  The included version of `MySql.data.dll` does not contain the method/plugin to make this work.  I was able to get the authentication to work by replacing the `MySql.data.dll` from the [MySql.Data NuGet Package](https://www.nuget.org/api/v2/package/MySql.Data/8.0.28) (I used the version from the net452 folder when testing).

```
NotSpecified: Exception calling "Open" with "0" argument(s): "Authentication method 'mysql_clear_password' not supported by any of the available plugins." 
```

**Comment by mithrandyr** [comment]
@twerthi please confirm that this is resolved in v1.9.0

**Comment by twerthi** [comment]
Apologies, I was out on vacation.  This has been tested and confirmed resolved.  Thanks!!
